j2cl-maven-plugin/src/main/java/com/vertispan/j2cl/mojo/BuildMojo.java

```diff
@@ -138,7 +139,7 @@ public class BuildMojo extends AbstractBuildMojo {
      * Closure flag: "Source of translated messages. Currently only supports XTB."
      */
     @Parameter
-    protected String translationsFile;
+    protected TranslationsFileConfig translationsFile;
```

Comment:
Ah got it. you use the two fields so that it makes sense to maven. What about just making this a Map<String, String> so that maven doesn't care what is there, or a dummy type to show it the "schema", but let the config read it out how it wants to?